src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsExecution.java

```diff
@@ -108,14 +107,14 @@ public boolean start() throws Exception {
 
     /**
      * TODO JAVADOC
+     *
      * @param workspaceId
      * @throws IOException
      */
     private void registerFingerprint(String workspaceId) throws IOException {
```

Comment:
Would `updateFingerprint` be a better name for this function (the fingerprint is already being registered in the `ExwsAllocateExecution` class)?